primo/data_parser/default_data.py

```diff
@@ -0,0 +1,268 @@
+        full_name="Federal Wetlands (Distant Range)",
+        is_submetric=True,
+        parent_metric="environment",
+        required_data="fed_wetlands_far",
+    ),
+    "state_wetlands_near": _SupportedContent(
+        name="state_wetlands_near",
+        full_name="State Wetlands (Close Range)",
+        is_submetric=True,
+        parent_metric="environment",
+        required_data="state_wetlands_near",
+    ),
+    "state_wetlands_far": _SupportedContent(
+        name="state_wetlands_far",
+        full_name="State Wetlands (Distant Range)",
+        is_submetric=True,
+        parent_metric="environment",
+        required_data="state_wetlands_far",
+    ),
+    # Submetrics of production_volume
+    "ann_production_volume": _SupportedContent(
+        name="ann_production_volume",
+        full_name="Annual Production Volume",
+        is_submetric=True,
+        parent_metric="production_volume",
+        required_data=[
+            "ann_gas_production",
+            "ann_oil_production",
+        ],
+    ),
+    "five_year_production_volume": _SupportedContent(
+        name="five_year_production_volume",
+        full_name="Five-year Production Volume",
+        is_submetric=True,
+        parent_metric="production_volume",
+        required_data=[
+            "five_year_gas_production",
+            "five_year_oil_production",
+        ],
+    ),
+    # Submetrics of other_emissions
+    "h2s_leak": _SupportedContent(
+        name="h2s_leak",
+        full_name="H2S Leak [Yes/No]",
+        is_submetric=True,
+        parent_metric="other_emissions",
+        required_data="h2s_leak",
+    ),
+    "brine_leak": _SupportedContent(
+        name="brine_leak",
+        full_name="Brine Leak [Yes/No]",
+        is_submetric=True,
+        parent_metric="other_emissions",
+        required_data="brine_leak",
+    ),
+}
+
+SUPP_EFF_METRICS = {
+    "num_wells": _SupportedContent(
+        name="num_wells",
+        full_name="Number of Wells",
+    ),
+    "num_unique_owners": _SupportedContent(
+        name="num_unique_owners",
+        full_name="Number of Unique Owners",
+    ),
+    "dist_centroid": _SupportedContent(
+        name="dist_centroid",
+        full_name="Distance to Centroid [miles]",
+    ),
+    "elevation_delta": _SupportedContent(
+        name="elevation_delta",
+        full_name="Average Elevation Delta [m]",
+    ),
+    "age_range": _SupportedContent(
+        name="age_range",
+        full_name="Age Range [Years]",
+    ),
+    "depth_range": _SupportedContent(
+        name="depth_range",
+        full_name="Depth Range [ft]",
+    ),
+    "avg_dist_road": _SupportedContent(
+        name="avg_dist_road",
+        full_name="Average Distance to Road [miles]",
+    ),
+    "avg_age": _SupportedContent(
+        name="avg_age",
+        full_name="Average Age [Years]",
+    ),
+    "avg_depth": _SupportedContent(
+        name="avg_depth",
+        full_name="Average Depth [ft]",
+    ),
+}
```

Comment:
Record completeness is an efficiency term we will need to consider here or add it as an update later at the very least.